src/engine/strat_engine/blockdev.rs

```diff
@@ -231,7 +248,7 @@ impl BlockDev {
         }
     }
 
-    pub fn wipe_metadata(&mut self) -> EngineResult<()> {
+    pub fn wipe_metadata(&self) -> EngineResult<()> {
```

Comment:
I dunno if that would be a good idea or not.

But it would definitely be correct to update ```StratPool::destroy()``` to accompany this change. ```self.block_devs.values_mut()``` can surely go to ```self.block_devs.values()```. Maybe ```StratPool::destroy()``` doesn't need to take a mutable reference either.